source/Symbol/SwiftASTContext.cpp

```diff
@@ -19,25 +19,26 @@
 #include <sstream>
 
 #include "swift/AST/ASTContext.h"
+#include "swift/AST/ASTMangler.h"
 #include "swift/AST/DebuggerClient.h"
 #include "swift/AST/Decl.h"
 #include "swift/AST/DiagnosticEngine.h"
 #include "swift/AST/ExistentialLayout.h"
-#include "swift/AST/IRGenOptions.h"
-#include "swift/AST/ASTMangler.h"
```

Comment:
Please submit the header sorting separately.

Reply:
Thanks for looking at this. I have no need to reorder the headers, it's just what git-clang-format did. So, I'll revise the PR to only add the new header.